project/buildSrc/src/main/kotlin/TokenGenerator.kt

```diff
@@ -0,0 +1,46 @@
+import java.security.MessageDigest
+import java.util.*
+
+class TokenGenerator {
+    companion object {
+        private const val STRING_SIZE = 256
+        private const val STRING_LENGTH = STRING_SIZE / 8
+
+        private const val HASH_ALGORITHM = "SHA-256"
+        private const val TOKEN_DURATION : Long = 1000 * 60 * 60 * 24 * 20 //Time in milliseconds before token expiring
+
+        private val charPool = ('a'..'z') + ('A'..'Z') + ('0'..'9') //List of chars used in the random string generation
+
+        fun generateRandomString() : String {
+            val randomString = (1..STRING_LENGTH)
+                    .map { kotlin.random.Random.nextInt(0, charPool.size) }
```

Comment:
It **might** not be a good idea to use this random source. See [this](https://stackoverflow.com/questions/11051205/difference-between-java-util-random-and-java-security-securerandom).

Reply:
Yes that makes sense, I'll update the code.